apps/jobbrowser/src/jobbrowser/templates/job_browser.mako

```diff
@@ -2444,13 +2444,16 @@ ${ commonheader("Job Browser", "jobbrowser", user, request) | n,unicode }
           <!-- ko with: coordinatorActions() -->
           <form class="form-inline">
             ##<input data-bind="value: textFilter" type="text" class="input-xlarge search-query" placeholder="${_('Filter by name')}">
```

Comment:
This is also a comment?

Reply:
Yeah, I'll leave that one in place for now. As a placeholder for where the free text name search would be if ever required.